src/spyglass/spikesorting/v1/recording.py

```diff
@@ -0,0 +1,730 @@
+from typing import Tuple, Iterable, Optional, Union, List
+import uuid
+
+import datajoint as dj
+import numpy as np
+import pynwb
+import probeinterface as pi
+import spikeinterface as si
+import spikeinterface.extractors as se
+from neuroconv.tools.spikeinterface.spikeinterfacerecordingdatachunkiterator import (
+    SpikeInterfaceRecordingDataChunkIterator,
+)
+from hdmf.data_utils import GenericDataChunkIterator
+
+from spyglass.common import Session  # noqa: F401
+from spyglass.common.common_ephys import Electrode, Raw  # noqa: F401
+from spyglass.common.common_device import Probe
+from spyglass.common.common_nwbfile import AnalysisNwbfile, Nwbfile
+from spyglass.common.common_interval import (
+    IntervalList,
+    interval_list_intersect,
+)
+from spyglass.common.common_lab import LabTeam
+from spyglass.utils.misc import generate_nwb_uuid
+
+schema = dj.schema("spikesorting_v1_recording")
+
+
+@schema
+class SortGroup(dj.Manual):
+    definition = """
+    # Set of electrodes to spike sort together
+    sort_group_id: str  # identifier for a group of electrodes
```

Comment:
Is there a benefit to having this a `str` instead of the `int` it was before?

Reply:
I thought making it `str` would enable more descriptive names, e.g. `'probe shank 1'` or `'tetrode 1'` seems better than `1`. But I don't have a strong opinion about this and I'm happy to change it back to int.

Reply:
I think it'll be easier to just have int in this case to keep consistency.

Reply:
I thought more about this and I would prefer to make this `varchar`. I think doing so is more consistent because `electrode_group_name` from `ElectrodeGroup` table is also `varchar`. Using a number to refer to an electrode group is not very descriptive.